### ISSUE TITLE
Documentation required to address Safari list-style affecting list role

### ISSUE THREAD

**Issue by sbeaumontweb** [bbc/gel#145]
Hullo all.

Whilst creating accessible components that use list elements styled with `list-style:none;`, we have noted that VO with Safari will no longer announce the element as a list, nor the elements within it as list items. It would appear that using this CSS property in Safari affects the role that the list has in the a11y tree. There are a couple of ways to get around this:

1. When using a list, add `role="list"` to the element. We found this to be confusing to come across as engineers, as at a quick glance it would seem redundant. We would be forced to leave comments in multiple places to explain why it was there.

2. In WebCore we have implemented a different method of removing the default list style in our reset stylesheet:

`ol, ul {
	list-style: none;
}
li:before {
	content: "\\200B";
	position: absolute;
} `

In short, we still remove the list style, but we then use a pseudo-element to force Safari into thinking that we're setting our own bullet style. In reality, `\\200B` is a zero-width space. The element is then announced as a list again.

A good example of where this happens is the Pagination component demo - the pages are marked up as a list, but not announced as such in VO + Safari.

It would be great to document a solution - even if we offer both the role and the CSS workaround as options for devs and let them make their own decision on which fits best in their codebase.

Any questions feel free to slack me. :) Thanks.
